src/chrome/content/rules/fujitsu-webmart.com.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	For other Fujitsu related rulesets, see Fujitsu.xml
+
+	Non-functional hosts in *.fujitsu-webmart.com:
+		Certificate mismatched:
+		- fujitsu-webmart.com
+		- www.fujitsu-webmart.com
+-->
+<ruleset name="Fujitsu (partial)" default_off="cert-invalid">
```

Comment:
`webmnt.fujitsu-webmart.com` was the only missing target I found

Reply:
added the missing target. thanks.